### ISSUE TITLE
[Feature Request] Choose which icons appear in sidebar

### ISSUE THREAD

**Issue by lichenophile** [sanoojes/spicetify-extensions#4]
It would be neat if I could either choose to continue displaying some icons/custom apps in the top area or perhaps make the divider between these icons and the library draggable so the icons can be in a smaller scrollable box to allow more library view.

Interestingly, my bookmarks custom-app has actually remained on top
![image](https://github.com/user-attachments/assets/8c8835b9-87f7-4051-a4ed-95a69b6b326f)

But as you can seee, I have a lot of apps that really obscure my access to playlists
![image](https://github.com/user-attachments/assets/f29ae525-3f82-45a0-8181-35d55847568c)

**Comment by sanoojes** [comment]
Yes, that's a great feature idea! I like your suggestion of changing icons. I'll try to add it soon, after my exams.
